src/ray/common/ray_config_def.h

```diff
@@ -768,19 +768,27 @@ RAY_CONFIG(int64_t, grpc_keepalive_time_ms, 10000)
 /// grpc keepalive timeout.
 RAY_CONFIG(int64_t, grpc_keepalive_timeout_ms, 20000)
 
-/// NOTE: we set a loose client keep alive because
+/// NOTE: We set a loose client keep alive because
 /// they have a failure model that considers network failures as component failures
 /// and this configuration break that assumption. We should apply to every other component
 /// after we change this failure assumption from code.
 /// grpc keepalive timeout for client.
 RAY_CONFIG(int64_t, grpc_client_keepalive_time_ms, 300000)
-
-/// grpc keepalive timeout for client.
+/// gRPC keep-alive timeout for client.
 RAY_CONFIG(int64_t, grpc_client_keepalive_timeout_ms, 120000)
-
+/// Number of keep-alive pings client can send before needing to send a data/header frame
+/// (0 indicates that an infinite number of pings can be sent without sending a data frame
+/// or header frame)
+RAY_CONFIG(int64_t, grpc_http2_max_pings_without_data, 0)
```

Comment:
Perhaps just use the descriptions from and links to the [canonical docs](https://grpc.github.io/grpc/core/md_doc_keepalive.html)?

Also, can we use the default values?

Reply:
These are mostly descriptions from gRPC docs with some clarifications.

> Also, can we use the default values?

Changing defaults here to make sure that we're able to detect disconnections irrespective of whether connection is actively used currently or not.